### ISSUE TITLE
armeabi-v7a Android device CURLOPT_HEADERFUNCTION failed.

### ISSUE THREAD

**Issue by dspec1984** [91Act/curl-unity#6]
Hello.
On the Android device of armeabi-v7a, `thiz 'is null in HeaderFunction and` size` and `nmemb` become negative values. Is there a workaround?

```
        [AOT.MonoPInvokeCallback(typeof(Delegates.HeaderFunction))]
        private static long HeaderFunction(IntPtr ptr, long size, long nmemb, IntPtr userdata)
        {
            size = size * nmemb;
            var thiz = ((GCHandle)userdata).Target as CurlEasy;
            ....
```

**Comment by Jayatubi** [comment]
Are you using the master version or the release version?

**Comment by dspec1984** [comment]
Yes, I am using the latest `master`.

**Comment by Jayatubi** [comment]
This should be fixed in 9230434d979623a2b805f0eaf77c5d0daf0dd00f. I've tested on the 32bit mono and 32/64 IL2CPP builds.

**Comment by dspec1984** [comment]
Thank you very much.

We were able to confirm normal operation here as well.
